lib/foods.js

```diff
@@ -1,5 +1,37 @@
+function persistFood(foodName, foodCalories) {
+  if (window.localStorage) {
```

Comment:
i believe this will always be true. `localStorage` is always going to exist so this is most likely an unnecessary check here 👍